dynamo/configuration.py

```diff
@@ -3,17 +3,100 @@
 from matplotlib import rcParams, cm, colors
 from cycler import cycler
 import matplotlib.pyplot as plt
+from .dynamo_logger import main_info
+
+
+class DynamoConfig:
+    """dynamo config class holding static variables to change behaviors of functions globally."""
+
+    # set the data store mode.
+    # saving memory or storing more results
+    # modes: full, succint, default
+    data_store_mode = None
+    keep_filtered_genes_default = None
+    keep_raw_layers_default = None
+    keep_filtered_cells_default = None
+    recipe_monocle_keep_filtered_genes_default = None
+    recipe_monocle_keep_filtered_cells_default = None
+    recipe_monocle_keep_raw_layers_default = None
+
+    del_2nd_moments = None
+
+    (
+        KEEP_FITLERED_CELLS_KEY,
+        KEEP_FILTERED_GENES_KEY,
+        KEEP_RAW_LAYERS_KEY,
+        RECIPE_MONOCLE_KEEP_FILTERED_CELLS_KEY,
+        RECIPE_MONOCLE_KEEP_FILTERED_GENES_KEY,
+        RECIPE_MONOCLE_KEEP_RAW_LAYERS_KEY,
```

Comment:
For each function, we can set FUNC_KEEP_HERE to control true/false default values via config. RECIPE_MONOCLE is for function def recipe_monocle, since it has different default values.

Reply:
what you respond here has nothing to do with what I am suggesting